docs/api/structures/referrer.md

```diff
@@ -0,0 +1,12 @@
+# Referrer Object
+
+* `url` String - HTTP Referrer URL.
+* `policy` String - Referrer Policy, as specified in the [Referrer-Policy
+  header][1].
+
+`policy` can be one of `default`, `unsafe-url`, `no-referrer-when-downgrade`,
```

Comment:
@MarshallOfSound do you remember if the docs linter will pick up allowable values in this format?

Reply:
It has to be inline in the description of the property.

So:

```markdown
* `policy` String - Can be `a`, `b`, `c` or `d`.  This part then describes policy
```